modules/MwEmbedSupport/mediawiki/mediawiki.client.js

```diff
@@ -82,7 +82,10 @@
 	mw.isIpad3 = function () {
 		return  /OS 3_/.test(userAgent) && mw.isIpad();
 	};
-	
+	mw.isIpadOnIos13 = function() {
+		return mw.isSafari() && mw.isTouchDevice();
```

Comment:
this will be true for iOS as well, no? and it will be correct also for 14, 15 an onwards so isIpadOS is a better name